index.js

```diff
@@ -17,6 +17,11 @@ const { convertOpenApiVersionToV3, getSpecByVersion, versions } = require('./lib
 const processors = require('./lib/processors');
 const listEndpoints = require('express-list-endpoints');
 
+const OPTIONS_METHOD = 'options';
```

Comment:
not sure if this is a related change but at this point it'd be more work to undo it, so might as well keep it; just an fyi for next time:D

Reply:
Totally unrelated, sorry 😄 I was thinking how to refactor to avoid the early return but in an elegant way (same test coverage thing). Forgot to rollback!

Reply:
Rollbacked!